changelog.txt

```diff
@@ -1,3 +1,19 @@
+# [v0.23.0] - 08-05-2024
+## What's Changed
+* build(deps): bump actions/checkout from 3 to 4 by @dependabot in https://github.com/kivikakk/comrak/pull/389
+* build(deps): bump once_cell from 1.17.0 to 1.19.0 by @dependabot in https://github.com/kivikakk/comrak/pull/390
+* build(deps): bump xdg from 2.4.1 to 2.5.2 by @dependabot in https://github.com/kivikakk/comrak/pull/391
+* build(deps): bump derive_builder from 0.12.0 to 0.20.0 by @dependabot in https://github.com/kivikakk/comrak/pull/392
+* build(deps): bump memchr from 2.5.0 to 2.7.2 by @dependabot in https://github.com/kivikakk/comrak/pull/396
+* build(deps): bump ntest from 0.9.0 to 0.9.2 by @dependabot in https://github.com/kivikakk/comrak/pull/397
+* build(deps): bump typed-arena from 2.0.1 to 2.0.2 by @dependabot in https://github.com/kivikakk/comrak/pull/398
+* Update automerge.yml by @gjtorikian in https://github.com/kivikakk/comrak/pull/401
+* build(deps): bump clap from 4.0.32 to 4.5.4 by @dependabot in https://github.com/kivikakk/comrak/pull/400
+* build(deps): bump regex from 1.7.0 to 1.10.4 by @dependabot in https://github.com/kivikakk/comrak/pull/402
+* Fix release workflows by @gjtorikian in https://github.com/kivikakk/comrak/pull/395
+
+
+**Full Changelog**: https://github.com/kivikakk/comrak/compare/v0.23.1-pre.3...v0.23.0
```

Comment:
1. Changelog is meant to inform the users of the software about notable changes. However, this list seems to include many internal changes that are of little interest to the end users. Perhaps it can be filtered somehow? You may also find this document interesting: https://keepachangelog.com
2. Many dependency version bumps. Has any consideration been given to MSRV? For example clap v4.5.4 requires Rust 1.74. What's the current MSRV policy of this project?

Reply:
Worth noting this isn't actually a real release — [0.23.0](https://crates.io/crates/comrak/0.23.0) was published a last week.

1. Perhaps indeed! I make no promises here, it depends on the release engineer's energy levels. Mine are usually pretty low.
2. `package.rust-version` currently says 1.62.1. I assume our CI workflow/Dependabot needs to be brought up to speed.